gradle/libs.versions.toml

```diff
@@ -64,7 +64,7 @@ qpidJmsClient = "2.5.0"
 slf4j = "2.0.13"
 testContainers = "1.19.7"
 typesafeConfig = "1.4.3"
-wiremock = "2.35.2"
+wiremock = "3.0.1"
```

Comment:
Note the [breaking changes in major version 3](https://github.com/wiremock/wiremock/releases/tag/3.0.0): The artifact name should also be changed from `wiremock-jre8-standalone` to `wiremock-standalone`.